scalameter-core/src/main/scala/org/scalameter/Measurer.scala

```diff
@@ -279,21 +308,22 @@ object Measurer {
    *  susceptible to actual randomness, because adding artificial noise increases
    *  the confidence intervals.
    */
-  trait Noise extends Measurer {
+  trait Noise extends Measurer[Double] {
```

Comment:
It's a part of Measurer still, but I think that the next step (when splitting `Measurer` to `Runner` and `Gauge` and enabling composition) should be to make it as a function that can be used in the `measurer.map`.
What do you think about it?